src/constants.py

```diff
@@ -11,20 +11,24 @@ class Signs:
 
 
 how_to_fix = "## <i>How to fix?</i>\n<b>The full Definition of Done of the AI-Gallery template and fix approached can be found [HERE]({detail_link}).</b>".format(
-    detail_link=details_help_link)
+    detail_link=details_help_link
```

Comment:
To many format change. is it because py version ?

Reply:
I use ruff to format the source code.